typescript/cli/src/consts.ts

```diff
@@ -1,4 +1,11 @@
 export const MINIMUM_CORE_DEPLOY_GAS = (1e8).toString();
 export const MINIMUM_WARP_DEPLOY_GAS = (1e7).toString();
 export const MINIMUM_TEST_SEND_GAS = (3e5).toString();
+
+
+export enum CRUD_COMMANDS {
+  DEPLOY = 'deploy',
+}
```

Comment:
Not sure this is the best place for this. Maybe something in the commands folder like we did for SIGN_COMMANDS

Reply:
fixed